provider/bluecat/bluecat.go

```diff
@@ -215,7 +215,7 @@ func (p *BluecatProvider) Records(ctx context.Context) (endpoints []*endpoint.En
 			tempEndpoint := endpoint.NewEndpoint(rec.Name, endpoint.RecordTypeTXT, rec.Properties)
 			tempEndpoint.Labels[endpoint.OwnerLabelKey], err = extractOwnerfromTXTRecord(rec.Properties)
```

Comment:
Since the only error being returned by `extractOwnerfromTXTRecord` is if an owner is not found, and not having an owner is a valid thing that can happen (ex if something was created in Bluecat by something other than External DNS), it might make sense to just not return an error on the function.

Another option could be to get the value and do not directly attempt to set the label of the Endpoint, and then if an error is not returned in the function, the endpoint would have the label added, and it would be added to the list of endpoints.

@seanmalloy what if your opinion on that?

Reply:
After talking with Sean, we think this is the appropriate way to handle this